pandas/tests/test_generic.py

```diff
@@ -824,10 +824,21 @@ def test_interp_scipy_basic(self):
 
     def test_interp_limit(self):
         s = Series([1, 3, np.nan, np.nan, np.nan, 11])
+
         expected = Series([1., 3., 5., 7., np.nan, 11.])
         result = s.interpolate(method='linear', limit=2)
         assert_series_equal(result, expected)
 
+        expected = Series([1., 3., 5., np.nan, 9., 11.])
+        result = s.interpolate(method='linear', limit=1,
+                               limit_direction='symmetric')
```

Comment:
do a ``limit_direction='forward'`` (same as the preivous test, and put a comment that this is for compat with current).